src/ResourceManagement/Billing/Billing.Tests/ScenarioTests/InvoicesTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Billing.Tests.Helpers;
+using Microsoft.Azure.Management.Billing;
+using Microsoft.Azure.Management.Billing.Models;
+using Microsoft.Azure.Management.Resources;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using System;
+using System.Linq;
+using System.Net;
+using Xunit;
+
+namespace Billing.Tests.ScenarioTests
+{
+    public class InvoicesTests : TestBase
+    {
+        const string DownloadUrlExpand = "downloadUrl";
+        const string RangeFilter = "invoicePeriodEndDate ge 2017-01-31 and invoicePeriodEndDate le 2017-02-28";
```

Comment:
@Panda-Wang is this still valid in sdk? If yes then the SDK and PS developer experience is different.
You are fine with this difference in functionality?